docs/serverless/08-02-expose-function.md

```diff
@@ -75,13 +75,13 @@ Follows these steps:
     ```bash
     curl https://$NAME.$DOMAIN
     ```
-   
-   >**NOTE:** If you are using Minikube, make sure that you've added the endpoint to the Minikube IP entry in the `/etc/hosts` file. 
+
+   >**NOTE:** If you are using Minikube, make sure that you've added the endpoint to the Minikube IP entry in the `/etc/hosts` file.
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="ui">
+    UI
```

Comment:
```suggestion
    Console UI
```
Refer to [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478304363).